improver_tests/acceptance/SHA256SUMS

```diff
@@ -577,34 +577,34 @@ e653ae271dd789dde8b03e6237cc7b48552dd9d5206d3430ee68d283683247a0  ./wind_downsca
-60dafc8944e39ee2798fb8a6bd681f02a3529327b1257d5061ca7d5a7ef30c7b  ./wxcode/basic/probability_of_lwe_precipitation_rate_above_threshold.nc
-1658a7001586a7be5820a0f10d648ecbbf09a559e04f08803dd8b2168bd0f9c1  ./wxcode/basic/probability_of_lwe_precipitation_rate_in_vicinity_above_threshold.nc
-aa3f3e25acd5c81b93b5a6385e6076043c802d9e6e2fbcff2a3341a9eb6a2681  ./wxcode/basic/probability_of_lwe_sleetfall_rate_above_threshold.nc
-4cda6b469e048d96635266db041e87eeb47535d41d8f333a66af00a2a68e835c  ./wxcode/basic/probability_of_lwe_snowfall_rate_above_threshold.nc
-2ffd4446ad9079b0bac2ae0de0c603be932f2f7f249a9274333fa2ae66b15393  ./wxcode/basic/probability_of_rainfall_rate_above_threshold.nc
-a839b8f7fa41edc856b70def5d38a5dc78d4d04c60a4354710d8fac12f1fa238  ./wxcode/basic/probability_of_texture_of_cloud_area_fraction_above_threshold.nc
-2d5d04f4417b4a6ab2a8bb8d2075c5d511b439020b4e6665d1e6c8b4f7cb5a41  ./wxcode/basic/probability_of_visibility_in_air_below_threshold.nc
-a6d2c7fcb9edb9b9240e735e40486ed75dde9a1aac3528dd16b1270fb7fa8c39  ./wxcode/global/kgo.nc
-3ed718f3646bffd2216d0028b03faf86f2fd90f2a570362cae08f05da22f87f0  ./wxcode/global/probability_of_cloud_area_fraction_above_threshold.nc
-eedbab4569285f89bd340461d5a41bffdb8095d851693c219ebf9b78b0540b1c  ./wxcode/global/probability_of_convective_ratio_above_threshold.nc
+2232c8c9462f80f0c02e8a216692e61ef04ab7f0f6a0c1eddaad1bf833a461b7  ./wxcode/basic/kgo.nc
+b04ad134a234d4af622d445e007c46fb8940f1b2176fd14131e701ac431cf82c  ./wxcode/basic/kgo_no_lightning.nc
+df2e494a05d8c7d50ed5db20e21b25e77e77064eeb128fc5c047854d1936d469  ./wxcode/basic/probability_of_cloud_area_fraction_above_threshold.nc
+da283b865cd7c3db7cbf130f3521f7d384b9fca6823341bb755bac28f991c301  ./wxcode/basic/probability_of_lightning_flashes_per_unit_area_in_vicinity_above_threshold.nc
+6e500004ec43c0328a4107675fc92c5b39528e13004b3204cade388b0b36504b  ./wxcode/basic/probability_of_low_type_cloud_area_fraction_above_threshold.nc
+d1c3259f8df4c8218ccbd5b817245100d6038cd5f26c0bd0dfa68aa3bc8b2183  ./wxcode/basic/probability_of_lwe_precipitation_rate_above_threshold.nc
+26cc88bfd3bc76ca4c4ea4ae73b4d398cc64aeb48a2d461275439580455293f8  ./wxcode/basic/probability_of_lwe_precipitation_rate_in_vicinity_above_threshold.nc
+c129c44f12d3d413cd6f67d8bb0a033b6e53af61add16f97e69e14a6b1af903a  ./wxcode/basic/probability_of_lwe_sleetfall_rate_above_threshold.nc
+10f12e254671c9ffc724242ae54c8b63c3f3bf38880fd03291e4978dda834c4c  ./wxcode/basic/probability_of_lwe_snowfall_rate_above_threshold.nc
+bedca029c7ba9719595683059945e355b48d9ae10964d5ef4e45842be25e10ff  ./wxcode/basic/probability_of_rainfall_rate_above_threshold.nc
+3b4aaa9dbcd4719257fdbb41ea4bab898f5bfc89d50e6e0ea841f9e010ae15a7  ./wxcode/basic/probability_of_texture_of_cloud_area_fraction_above_threshold.nc
+a6b9f46daa925737cf8851be89a87a4787230dff0758adf8091b7d7cabf59747  ./wxcode/basic/probability_of_visibility_in_air_below_threshold.nc
```

Comment:
I see you've changed the `mosg__model_configuration` attribute for this file from `"uk_det uk_ens"` to be `"uk_ens"` and it looks like you've done something similar on other input files. I don't think this is appropriate as it doesn't reflect the input data we will actually be using. Please can you explain this?

Reply:
This was due to my misunderstanding of what inputs were expected. I've now reverted these changes.